backend/src/main/kotlin/uk/gov/justice/digital/backend/client/preview/AthenaPreviewClient.kt

```diff
@@ -25,10 +25,14 @@ class AthenaPreviewClient(@Named("preview") private val previewDataSource: DataS
         val columnNames = (1..columnCount)
             .map { metadata.getColumnName(it) }
 
-        val result = mutableListOf<Map<String, String>>()
+        // First row of result contains the column names
+        val result = mutableListOf<List<String>>(columnNames)
 
         while(resultSet.next()) {
-            result.add( columnNames.associateWith { resultSet.getString(it) } )
+            val row = (1..columnCount)
+                .map { resultSet.getString(it) }
```

Comment:
Will all the data types always be of type String or can there be cases of other data types as well?

Reply:
It will always be a string.